src/components/EditableTagField/6-EditableTagField.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from "react";
+import { EditableTagField } from "./index";
+import "antd/es/select/style/css";
+import "antd/es/carousel/style/css";
+
+const instanceTags = [
```

Comment:
Is this being used? It looks like you declare this again on line 18 in src/components/EditableTagField/EditableTagField.test.tsx

Reply:
Yeah they are both being used in their own use cases.This is being used in the storybook story for this component. Which can be viewed by running `yarn storybook` and the other is being used in the jest tests for this component.